idea_town/frontend/static-src/styles/modules/_avatar.scss

```diff
@@ -1,18 +0,0 @@
-#avatar-wrapper {
```

Comment:
FWIW, I think we might still need this stuff for the contributors list in the experiment details page